src/components/date/date.spec.tsx

```diff
@@ -630,10 +741,10 @@ describe("Date", () => {
         simulateFocusOnInput(wrapper);
         act(() => {
           wrapper
-            .update()
+            ?.update()
             .find(DatePicker)
-            .props()
-            .onDayClick(mockDate, {}, { target: {} });
+            ?.props()
+            ?.onDayClick?.(mockDate, mockEvent);
```

Comment:
```suggestion
            ?.onDayClick?.(mockDate, mockEvent);
```
```suggestion
            .onDayClick?.(mockDate, mockEvent);
```

**suggestion** (non-blocking): think you got carried away with optional chaining here, as this one doesn't seem to be necessary - `props` always returns a props object